bin/hosts.js

```diff
@@ -67,13 +94,28 @@ exports.remove = function (ip, host) {
 	exports.writeFile(lines);
 };
 
+exports.removeMany = function (hostList) {
+  var lines = exports.get(), unchanged = true;
+  
+  // Try to remove entries, if they exist
+  for (let i = 0, l = hostList.length; i < l; i++) {
+    let host = hostList[i];
+    lines = lines.filter(function (line) {
+      return unchanged=!(Array.isArray(line) && line[0] === host[0] && line[1] === host[1]);
```

Comment:
`unchanged=!` is not easy to read. This is an assignment inside a `return` statement, which is pretty much never a good idea. 
Besides, you override `unchanged` with the last array entry's result. 

Unfortunately, JS doesn't have a quick method to compare an array (because why, duh), but the following should work:

```js
exports.removeMany = function (hostList) {
  var lines = exports.get(), changed = true;

  // Try to remove entries, if they exist
  for (let i = 0, l = hostList.length; i < l; i++) {
    let host = hostList[i];
    const new_lines = lines.filter((line) => !(Array.isArray(line) && line[0] === host[0] && line[1] === host[1]));
    changed |= lines.length !== new_lines.length;
    lines = new_lines;
  }

  if (changed) { exports.writeFile(lines); }
};

```